controller/documentcontroller.php

```diff
@@ -547,6 +547,9 @@ public function wopiCheckFileInfo($fileId){
 			return false;
 		}
 
+		$instanceId = \OCP\Config::getSystemValue('instanceid', '');
+		$instanceId = substr(md5($instanceId), -13); // Don't expose the secret ID.
```

Comment:
Okay, though I think there is no harm in exposing the InstanceId as it is, but no harm this way too.